main.go

```diff
@@ -50,11 +52,41 @@ var tsclient *tailscale.LocalClient
 func main() {
 	flag.Parse()
 
-	log.Println("Verifying we can fetch machines from Mosyle...")
-	_, err := enumerateMachines()
+	log.Println("Logging in to Mosyle ...")
+	err := mosyleLogin()
+	if err != nil {
+		log.Fatalf("failed to login to mosyle: %v", err)
+	}
+
+	go func() {
+	DailyRefresh:
+		for {
+			time.Sleep(23 * time.Hour)
```

Comment:
Would be nice to get more than 1 hour of leeway, but maybe it doesn't matter

```suggestion
			time.Sleep(20 * time.Hour)
```